### DISCUSSION TITLE
a limit?

### DISCUSSION THREAD

**Discussion by legaltextai** [freelawproject/courtlistener#3115] [Q&A] [answered]
I am trying to download all opinions from scotus. https://www.courtlistener.com/api/rest/v3/opinions/?cluster__docket__court__id=scotus 
but i can only download the first 500 opinions, followed by error 502 
i presume this is a limit per day or per download?

**Comment by mlissner** [comment, marked as answer]
Unfortunately, if you paginate too far, we have to cut you off, so this could be that. See here: https://www.courtlistener.com/help/api/rest/#performance-tips
